test/unit/storage/entities/account.js

```diff
@@ -180,6 +180,14 @@ describe('Account', () => {
 			'fees',
 			'rewards',
 			'vote',
+			'u_username',
+			'u_isDelegate',
+			'u_secondSignature',
+			'u_nameExist',
+			'u_multiMin',
+			'u_multiLifetime',
+			'u_balance',
+			'productivity',
```

Comment:
returning productivity for every account?

Reply:
It's used for delegate accounts. It was already present in `extended` accounts but as the calculation is not costly I decide to move to `basic` accounts.